src/stan/variational/advi.hpp

```diff
@@ -518,7 +520,9 @@ namespace stan {
                            true, true, &msg);
         if (msg.str().length() > 0)
           logger.info(msg);
-        values.insert(values.begin(), 0);
+        values.insert(values.begin(), 0);  // the first line of lp__
```

Comment:
Question: are `log_p` and `log_q` supposed to be 0 here?

Optional: you could replace this with:
```
values.insert(values.begin(), {0, 0, 0});
```

If you think it'll look any cleaner.